src/row.js

```diff
@@ -285,6 +285,7 @@ Row.prototype.create = function(options, callback) {
  * @param {object|object[]} rules The rules to apply to this row.
  * @param {object} [gaxOptions] Request configuration options, outlined here:
```

Comment:
We should convert this to an `options` object:

- [options]
- [options.gaxOptions]
- [options.appProfileId]